coil/src/main/java/dev/chrisbanes/accompanist/coil/Coil.kt

```diff
@@ -135,65 +122,90 @@ fun CoilImage(
     loading: @Composable (() -> Unit)? = null,
     onRequestCompleted: (RequestResult) -> Unit = emptySuccessLambda
 ) {
-    WithConstraints(modifier) {
-        val requestWidth = constraints.requestWidth.value
-        val requestHeight = constraints.requestHeight.value
+    var result by state<RequestResult?> { null }
+    val callback = mutableStateOf(onRequestCompleted)
```

Comment:
Why is this a state? this value is never changed and `onRequestCompleted` is defined as a param

Reply:
This was something @adamp mentioned in https://github.com/chrisbanes/accompanist/pull/30#discussion_r441850532